website/src/components/SimpleTable/SimpleTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { Table, TableContainer, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
+
+interface TableProps {
+  headers: string[];
+  rows: string[][];
+}
+// Create a SimpleTable component that takes in a list of headers and a list of rows
+export const SimpleTable = ({ headers, rows }: TableProps) => {
```

Comment:
I saw it, but it looked very complicated, with many options that are not required in this case. But I understand that creating a whole new component just for this is not a good idea. I will try to use the Table form chakra directly.

Reply:
In the end, I used the DataTable component.

Reply:
Although I was able to make it work, I think that the code I wrote in the `getDataTable()` function looks awful. I have the feeling that it could be done without having to repeat the name of the states in so many places (`interestingStates`, `LangRow` and in the column helpers). Do you see a better approach?